com.polyphonesoundfonts.polyphone.json

```diff
@@ -0,0 +1,156 @@
+{
+  "app-id": "com.polyphonesoundfonts.polyphone",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.14",
+  "sdk": "org.kde.Sdk",
+  "command": "polyphone",
+  "rename-desktop-file": "polyphone.desktop",
+  "rename-icon": "polyphone",
+  "finish-args": [
+    /* X11 + XShm access */
+    "--share=ipc",
+    "--socket=x11",
+    /* Wayland */
+    /* "--socket=wayland", */
```

Comment:
Just drop it if you don't need it.

Reply:
I forget why it's commented out but it works with it in. So I'll add it back.